pkg/cloudprovider/providers/aws/instances.go

```diff
@@ -19,15 +19,15 @@ package aws
 import (
 	"fmt"
 	"net/url"
+	"regexp"
 	"strings"
+	"sync"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/golang/glog"
 	"k8s.io/api/core/v1"
```

Comment:
Normally we will group `k8s.io` packages together, keeping them separate from other non k8s.io packages. Right?